src/app/scripts/cac/pages/cac-pages-directions.js

```diff
@@ -1,12 +1,21 @@
 CAC.Pages.Directions = (function ($, _, DirectionsList, Itinerary, Settings, Utils) {
     'use strict';
 
+    var center = [39.95, -75.1667];
+    var zoom = 12;
+    var cartodbAttribution = [
```

Comment:
Should probably comment a TODO here about the stuff duplicated from the map controller.